substrate/frame/identity/src/lib.rs

```diff
@@ -998,4 +998,23 @@ impl<T: Config> Pallet<T> {
 		IdentityOf::<T>::get(who)
 			.map_or(false, |registration| (registration.info.fields().0.bits() & fields) == fields)
 	}
+
+	/// Set identity for some account
+	pub fn set_identity_no_deposit(
```

Comment:
Is there really no way to do a migration without editing identity pallet with a new function?
How callable is this for users ?

Reply:
> Is there really no way to do a migration without editing identity pallet with a new function?

There is. We could define a storage alias for the `IdentityOf` in the migration and manipulate it directly. This way was just easier and I don't see a good reason to not extend identity pallet like this. Especially that it's a temporary extension - we can remove this function together with the migration after it's executed on all live networks.

> How callable is this for users ?

It's not. It's only available from inside the runtime, it's not a pallet call.